sdk/communication/communication-phone-numbers/test/public/siprouting/utils/msUserAgentPolicy.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  PipelinePolicy,
+  PipelineRequest,
+  PipelineResponse,
+  SendRequest,
+} from "@azure/core-rest-pipeline";
+import { env } from "@azure-tools/test-recorder";
+
+export function createMSUserAgentPolicy(): PipelinePolicy {
+  return {
+    name: "msUserAgentPolicy",
+    async sendRequest(request: PipelineRequest, next: SendRequest): Promise<PipelineResponse> {
+      const useragent = env.AZURE_USERAGENT_OVERRIDE;
+
+      request.headers.set("x-ms-useragent", "acs-test");
```

Comment:
I am not clear on this change, why are we setting default header value? Wouldn't it be better to just set environment variable AZURE_USERAGENT_OVERRIDE?
Btw. for live runs in pipeline, this variable is retrieved from kevault, and I believe it already has this value.

Reply:
See above comment.